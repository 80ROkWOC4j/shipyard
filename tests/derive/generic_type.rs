use shipyard::*;

#[system(Test)]
fn run<T>() {}

fn main() {}
