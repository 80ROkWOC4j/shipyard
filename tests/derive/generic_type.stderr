error: run should not take generic arguments
 --> $DIR/generic_type.rs:4:7
  |
4 | fn run<T>() {}
  |       ^^^
